docs/cli/operator.mdx

```diff
@@ -564,6 +571,367 @@ Use the options listed below to change the command's behavior.
 
 - [--visibility-uri](/cli/cmd-options#visibility-uri)
 
+## nexus
+
+Nexus commands enable managing Nexus resources.
+
+Nexus commands follow this syntax:
+
+```
+temporal operator nexus [command] [command] [command options]
```

Comment:
Normally [command] [subcommand], not [command] [command]. Also does this use subcommands? I'd also remove "command" before options:

```suggestion
temporal operator nexus [command] [subcommand] [options]
```